src/workers/postScoutMatchedSlack.ts

```diff
@@ -0,0 +1,49 @@
+import { messageToJson, Worker } from './worker';
+import { Post } from '../entity';
+import { getDiscussionLink, webhook } from '../common';
+import { ChangeObject } from '../types';
+
+interface Data {
+  post: ChangeObject<Post>;
+}
+
+const worker: Worker = {
+  subscription: 'post-scout-matched-slack',
+  handler: async (message, _, logger): Promise<void> => {
+    const data: Data = messageToJson(message);
+    const { post } = data;
+    try {
+      if (!post.scoutId) {
```

Comment:
Locally getting the following error:
`"Cannot read properties of undefined (reading 'scoutId')",`

Reply:
Have you applied the latest migration?